src/Sylius/Behat/Resources/config/suites/api/order/modifying_address.yaml

```
@@ -0,0 +1,53 @@
+# This file is part of the Sylius package.
+# (c) Sylius Sp. z o.o.
+
+default:
+    suites:
+        api_modifying_address:
```

Should it somehow specify, that it's only about order address or if it's about all addresses, then be part of different suite?

```suggestion
        api_order_modifying_address:
```